ministryofjustice/laa-apply-for-criminal-legal-aid
# CRIMAP-673 add 11th tranch of providers

## Description of change
Add 11th tranch of providers

## Link to relevant ticket
[CRIMAP-673](https://dsdmoj.atlassian.net/browse/CRIMAP-673)

## Notes for reviewer

## Screenshots of changes (if applicable)

### Before changes:

### After changes:

## How to manually test the feature

Confirm against ticket.


[CRIMAP-673]: https://dsdmoj.atlassian.net/browse/CRIMAP-673?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
